sdk/storage/storage-internal-avro/package.json

```diff
@@ -12,86 +12,105 @@
     "build:es6": "tsc -p tsconfig.json",
     "build:samples": "echo Skipped.",
     "build:test": "npm run build:es6 && dev-tool run bundle",
     "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "dev-tool run vendored rimraf --glob dist dist-* types temp statistics.html coverage coverage-browser .nyc_output *.tgz *.log test*.xml TEST*.xml",
     "docs": "echo Skipped.",
-    "extract-api": "tsc -p . && dev-tool run extract-api",
+    "extract-api": "dev-tool run build-package && dev-tool run extract-api",
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "dev-tool run test:node-js-input --no-test-proxy -- -t 300000  \"dist-esm/test/*.spec.js\" \"dist-esm/test/node/*.spec.js\"",
+    "integration-test:node": "dev-tool run test:node-js-input --no-test-proxy -- --exclude 'test/snippets.spec.ts' -t 300000  \"dist-esm/test/*.spec.js\" \"dist-esm/test/node/*.spec.js\"",
     "lint": "eslint package.json api-extractor.json src test",
     "lint:fix": "eslint package.json api-extractor.json src test --fix",
     "pack": "npm pack 2>&1",
     "test": "npm run clean && npm run build:test && npm run unit-test",
     "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
     "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser",
-    "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "dev-tool run test:node-ts-input --no-test-proxy",
-    "update-snippets": "echo skipped"
+    "unit-test:browser": "npm run clean && dev-tool run build-package && dev-tool run build-test && dev-tool run test:vitest --browser",
+    "unit-test:node": "dev-tool run test:vitest",
+    "update-snippets": "dev-tool run update-snippets"
   },
   "dependencies": {
     "@azure/abort-controller": "^2.1.2",
-    "@azure/core-paging": "^1.1.1",
-    "tslib": "^2.2.0"
+    "@azure/core-paging": "^1.6.2",
+    "tslib": "^2.8.1"
   },
   "devDependencies": {
+    "@azure-tools/test-utils-vitest": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@types/chai": "^4.1.6",
-    "@types/mocha": "^10.0.0",
+    "@azure/logger": "^1.1.4",
     "@types/node": "^18.0.0",
-    "chai": "^4.2.0",
+    "@vitest/browser": "^3.0.6",
+    "@vitest/coverage-istanbul": "^3.0.6",
     "dotenv": "^16.0.0",
     "es6-promise": "^4.2.5",
     "eslint": "^9.9.0",
     "inherits": "^2.0.3",
```

Comment:
is this needed?

Reply:
Yes, as there were unit tests runners for the browser, so there may be as well here.

Reply:
My comment is on line 71, `inherits`